dolphinscheduler-alert/src/main/java/org/apache/dolphinscheduler/alert/AlertServer.java

```diff
@@ -129,9 +134,20 @@ private void runSender() {
             if (alertPluginManager == null || alertPluginManager.getAlertChannelMap().size() == 0) {
                 logger.warn("No Alert Plugin . Can not send alert info. ");
             } else {
-                List<Alert> alerts = alertDao.listWaitExecutionAlert();
-                alertSender = new AlertSender(alerts, alertDao, alertPluginManager);
-                alertSender.run();
+                InterProcessMutex mutex = null;
+                try {
+                    logger.error("创建分布式锁 : ");
```

Comment:
Please use English. If there is an abnormality (such as ZK dead), I think we still need to perform an alarm task (even if there will be repeated alarm messages)

Reply:
fix log message